ui/stories/toolbar/toolbar-link.md

```diff
@@ -0,0 +1,30 @@
+<!--THIS FILE IS AUTO GENERATED. This file is generated from JSDoc comments in app/components/auth-config-form/config.js. To make changes, first edit that file and run "yarn gen-story-md auth-config-form/config" to re-generate the content.-->
+
+## ToolbarLink
+`ToolbarLink` styles links for the Toolbar.
+
+**Properties**
+
+| Name | Type | Default | Description |
+| --- | --- | --- | --- |
+| params | <code>Array</code> | <code></code> | Array top pass to LinkTo params |
```

Comment:
I didn't see the typo in the component, so maybe this needs to be regenerated? `Array top pass` should be `Array to pass`